EO.l10n

```diff
@@ -309,8 +309,8 @@ core-sleep         dormu
 #core-slurp        slurp
 #core-snip         snip
 #core-snitch       snitch
-#core-so           so
-core-sort         ordigu
+core-so            ja
+core-sort          ordigu
```

Comment:
In that case, it feels like `ordige` is a better fit, as it creates a separate list, the original one is left unordered, so it’s not commanded to be sorted as the imperative form would rather imply.

That is:
```
my $tabelo = (2, 1)
$table.sort # (1, 2)
$tabelo # (2, 1)
$tabelo .= sort  # asignu sortige
$tabelo # (1, 2)
```